install/kubernetes/cilium/Chart.yaml

```diff
@@ -143,3 +143,11 @@ annotations:
       displayName: Cilium Pod IP Pool
       description: |
         CiliumPodIPPool defines an IP pool that can be used for pooled IPAM (i.e. the multi-pool IPAM mode).
+    - kind: CiliumWorldCIDRSet
+      version: v2alpha1
+      name: ciliumworldcidrsets.cilium.io
+      displayName: Cilium World CIDR Set
+      description: |
+        WorldCIDRs is a list of IPv4 CIDRs that will be considered outside the
+        "high-scale ipcache domain" (typically outside the clustermesh) and for
+        which traffic won't be encapsulated.
```

Comment:
I think this isn't entirely accurate because the user still must specify in the CRD whether encap is true or false, even though false is the only supported now.

Reply:
agreed, I'll reword.